fastify3/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "@contrast/fastify3-test-bench",
+  "private": true,
+  "version": "3.16.0",
+  "description": "Intentionally vulnerable Fastify3 application",
+  "author": "Contrast Security <nodejs@contrastsecurity.com>",
+  "license": "UNLICENSED",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Contrast-Security-OSS/NodeTestBenches.git"
+  },
+  "engines": {
+    "node": ">=10.13.0"
+  },
+  "bugs": {
+    "url": "https://github.com/Contrast-Security-OSS/NodeTestBenches/issues"
+  },
+  "homepage": "https://github.com/Contrast-Security-OSS/NodeTestBenches/tree/main/fastify3#readme",
+  "main": "server.js",
+  "scripts": {
+    "contrast": "DEBUG=contrast:* node-contrast server.js",
+    "dev": "nodemon server.js | pino-pretty",
+    "lint": "eslint --fix .",
+    "start": "node server.js | pino-pretty"
+  },
+  "dependencies": {
+    "@contrast/test-bench-utils": "^3.16.0",
+    "ejs": "2.6.1",
+    "fastify": "^3.9.2",
+    "fastify-cookie": "^3.6.0",
```

Comment:
I think `fastify-cookie` needs to be at `^5.1.0`, for fastify3. Otherwise There's a deprecation warning:
```
(node:420385) [FSTDEP006] FastifyDeprecation: You are decorating Requ
est/Reply with a reference type. This reference is shared amongst all
 requests. Use onRequest hook instead. Property: cookies
```

Reply:
Bumped and tested